src/components/shared/PaymentBankData.vue

```diff
@@ -163,6 +162,10 @@ export default Vue.extend( {
 				);
 			}
 		},
+		setAccountId: function ( accountId: string ) {
+			this.$props.accountId = accountId;
+			this.validate();
```

Comment:
Calling `validate` immediately will show an error message while the user is still typing (a behavior that always annoys me: "Yes, I know it's invalid, I'm still typing"). It'll also bombard our server endpoint with incomplete numbers.
I think we should only validate on blur and and when the form is submitted

Reply:
Oops, I had fixed that. I must have ctrl-z'd it back into existence